src/components/utils/linkedin-login.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const React = require('react-native');
+const {
+    Platform,
+    DeviceEventEmitter,
+    NativeModules
+} = React;
+const RNLinkedinLogin = NativeModules.LinkedinLogin;
+
+class LinkedinLogin {
+
+    _accessToken = null;
+    _expiresOn = null;
+
+    _redirectUrl = null;
+    _clientId = null;
+    _clientSecret = null;
+    _state = null;
+    _scopes = null;
+
+    constructor() {
+    }
+
+    /**
+     * Initializes the LinkedinLogin API
+     * @param  {string} redirectUrl     [description]
+     * @param  {string} clientId        [description]
+     * @param  {string} clientSecret    [description]
+     * @param  {string} state           [description]
+     * @param  {array} scopes           [description]
+     * @return {object} promise         [description]
+     */
+    init(redirectUrl, clientId, clientSecret, state, scopes) {
+        this._redirectUrl = redirectUrl;
+        this._clientId = clientId;
+        this._clientSecret = clientSecret;
+        this._state = state;
+        this._scopes = scopes;
+    }
+
+    /**
+     * Gets the Profile image
+     * @return {object} Returns the promise with the image
+     */
+    getProfileImages() {
+        const atoken = this._accessToken;
+
+        return new Promise((resolve, reject) => {
+            DeviceEventEmitter.addListener('linkedinGetRequest', (d) => {
+                const data = JSON.parse(d.data);
+
+                if (data.values) {
+                    console.log(data.values);
+                    resolve(data.values);
+                } else {
+                    reject('No profile image found');
+                }
+            });
+
+            DeviceEventEmitter.addListener('linkedinGetRequestError', (error) => {
+                reject(error);
+            });
+
+            const picstr = 'https://api.linkedin.com/v1/people/~/picture-urls::(original)';
+            const picstrWithAuth = `${picstr}?oauth2_access_token=${atoken}&format=json`;
+
+            if (Platform.OS === 'android') {
+                RNLinkedinLogin.getRequest(picstr);
+            } else {
+                // if ios
```

Comment:
Move this comment next to  "} else {".

In fact, let's check explicitly if Platform.OS === 'ios'. The platform can be anything; if react native starts supporting Windows OS, our code will break as soon as we update react native version.